models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1621,7 +1621,9 @@ FROM
     ('gnx-genaro-network', 'ethereum', 'GNX', 0x6ec8a24cabdc339a06a172f8223ea557055adaa5, 9),
     ('pndc-pond-coin', 'ethereum', 'PNDC', 0x423f4e6138e475d85cf7ea071ac92097ed631eea, 18),
     ('crvusd-curvefi-usd-stablecoin', 'ethereum', 'CRVUSD', 0xf939e0a03fb07f59a73314e73794be0e57ac1b4e, 18),
-    ('mav-maverick-token', 'ethereum', 'MAV', 0x7448c7456a97769f6cd04f1e83a4a23ccdc46abd, 18)
+    ('mav-maverick-token', 'ethereum', 'MAV', 0x7448c7456a97769f6cd04f1e83a4a23ccdc46abd, 18),
+    ('lcny-alternity-cny', 'ethereum', 'LCNY', 0x7635b612792e4bfb7f2fa12a3e5d5a3f2e3c34bc, 18),
+    ('altr-alternity', 'ethereum', 'ALTR', 0xD1ffCacFc630CE68d3cd3369F5db829a3ed01fE2, 18)
```

Comment:
These two tokens' price feeds are currently inactive.

Reply:
for reference @smolfork 
https://api.coinpaprika.com/v1/coins/lcny-alternity-cny
https://api.coinpaprika.com/v1/coins/altr-alternity

both return a result, but both have `"is_active":false` which means no continual prices will be added

what is the use case for these?

Reply:
you could contact coinpaprika directly about both of these to reactivate, if needed

Reply:
Hey, this is Radek from [@Coinpaprika](https://github.com/coinpaprika/). Unfortunately, currently, there is no volume for these assets on exchanges; thus, they are inactive (we can't calculate the price)

Reply:
awesome -- thanks for jumping in & providing context, much appreciated!

Reply:
@smolfork due to above comment, we will need to close this PR, as we won't be able to obtain prices for these tokens.